Lesson_10/wineshop_dyn_amounts.html

```diff
@@ -0,0 +1,189 @@
+        <button id="showShoppingCartBtn">Show shopping cart</button>
+        <div id="shoppingCart">
+        </div>
+        <h3>Total Price</h3>
+        <div id="total"></div>
+    </div>
+
+    <script>
+        let wines = [
+            {
+                title: 'Primitivo',
+                image: 'https://tmarchettico.com/wp-content/uploads/Verosso.jpg',
+                type: 'red',
+                origin: { country: 'Italy', region: 'Salento', winery: 'Verosso' },
+                year: 2016,
+                grapes: 'primitivo',
+                price: 4.34,
+                currency: '€',
+                rating: 3,
+                amount: 10
+            },
+            {
+                title: 'Chianti',
+                image: 'https://images.vivino.com/thumbs/_qpGWd60REu70cwIMqQHSw_pb_x600.png',
+                type: 'red',
+                origin: { country: 'Italy', region: 'Chianti', winery: 'Piccini' },
+                year: 2017,
+                grapes: 'Cabernet Sauvignon, Merlot, Sangiovese, Tempranillo, Ciliegiolo',
+                price: 11.82,
+                currency: '€',
+                rating: 4,
+                amount: 3
+            },
+            {
+
+                title: 'Amarone della Valpolicella Classico',
+                image: 'https://images.vivino.com/thumbs/WjrUTKpXSM2j7A82Xkkh7g_pb_x600.png',
+                type: 'red',
+                origin: { country: 'Italy', region: 'Amarone della Valpolicella Classico', winery: 'Tommasi' },
+                year: 2015,
+                grapes: 'Rondinella, Molinara, Corvinone, Corvina Veronese',
+                price: 35.90,
+                currency: '€',
+                rating: 5,
+                amount: 1
+            },
+            {
+                title: 'Brut (Carte Jaune) Champagne',
+                image: 'https://images.vivino.com/thumbs/s2PDIn8CQm-hAVb8u00QQw_pb_x600.png',
+                type: 'sparkling',
+                origin: { country: 'France', region: 'Champagne', winery: 'Veuve Clicquot' },
+                year: 2017,
+                grapes: 'Chardonnay, Pinot Noir, Pinot Meunier',
+                price: 44.73,
+                currency: '€',
+                rating: 5,
+                amount: 0
+            },
+            {
+                title: 'Oberhäuser Brücke Riesling Spätlese 2015',
+                image: 'https://cdn.vinello.de/media/image/eb/63/0b/doennhoff-oberhauser-brucke-riesling.png',
+                type: 'white',
+                origin: { country: 'Germany', region: 'Nahe', winery: 'Dönnhoff' },
+                year: 2015,
+                grapes: 'Riesling',
+                price: 26.50,
+                currency: '€',
+                rating: 4,
+                amount: 2
+            },
+        ]
+        let shoppingCart = [];
+
+        for (let i = 0; i < wines.length; i++) {
+            const wineListEl = document.getElementById('wineList');
+            if (wines[i].rating < 0) {
```

Comment:
Those checks right here is not really needed. You would need something like that of you don't control the incoming data. But you can keep it as a reference for future ofc)